src/main/java/de/tum/in/www1/artemis/service/LectureUnitProcessingService.java

```diff
@@ -98,10 +99,15 @@ public List<AttachmentUnit> splitAndSaveUnits(LectureUnitInformationDTO lectureU
     /**
      * Removes the break slides or solution slides from the given document.
      *
-     * @param document document to remove break slides from
+     * @param document                    document to remove break slides from
+     * @param removeBreakSlides           true, if the break slides should be removed
+     * @param removeBreakSlidesKeyword    the keyword that identifies a break slide
+     * @param removeSolutionSlides        true, if the example solution slides should be removed
+     * @param removeSolutionSlidesKeyword the keyword that identifies a example solution slide
      */
-    private void removeBreakOrSolutionSlides(PDDocument document, boolean removeBreakSlides, boolean removeSolutionSlides) {
-
+    private void removeBreakOrSolutionSlides(PDDocument document, boolean removeBreakSlides, String removeBreakSlidesKeyword, boolean removeSolutionSlides,
+            String removeSolutionSlidesKeyword) {
+        log.warn(removeBreakSlidesKeyword);
```

Comment:
If you want to keep the log, you should a descriptive message. Otherwise, no one will figure out what this log means.

Reply:
It was by mistake 83b85c1f7afc396d860a8baffedb028a960925f2